aten/src/ATen/core/Formatting.cpp

```diff
@@ -292,6 +292,11 @@ std::ostream& print(std::ostream& stream, const Tensor & tensor_, int64_t linesi
         stream << ", axis: " << tensor_.q_per_channel_axis();
       }
     }
+
+    auto fw_grad = tensor.fw_grad(/* level */ 0);
```

Comment:
Hmm, is unconditionally printing at level zero correct? Seems to me like you should print all levels, no?

Reply:
This first version only ever has 1 level. So the places where we will handle multiple levels in the future have 0 hardcoded for now in printing in python/c++ and evaluation of forward grad for example.